Sources/Networking/OfferingsAPI.swift

```diff
@@ -36,7 +38,9 @@ class OfferingsAPI {
         let offeringsCallback = OfferingsCallback(cacheKey: getOfferingsOperation.cacheKey, completion: completion)
         let cacheStatus = self.offeringsCallbacksCache.add(callback: offeringsCallback)
 
-        self.backendConfig.operationQueue.addCacheableOperation(getOfferingsOperation, cacheStatus: cacheStatus)
+        self.backendConfig.addCacheableOperation(getOfferingsOperation,
```

Comment:
Instead of adding it directly to the queue, a new method allows to introduce a random delay, but only *after* the callback is in the cache.

Reply:
Ahhhh I see, yeah, that's a much better way of doing it.